docs/operator-guide/server-setup.md

```diff
@@ -420,3 +431,71 @@ Another common failure mode is to have the joysetup.sh script fail due to a
 pre-existing zpool on the disks. When this happens often the server-setup
 plows forward and the agentsetup fails. If you see a failure, it's always a
 good idea to check the status of the zpool early in the debugging process.
+
+## iPXE booting Compute Nodes
+
+In some situations, such as a Bare-Metal-as-a-Service (BMaaS) provider, they
+may have instances that by default only network boot on one network
+interface, and it may not be the one a Triton installation has configured as
+the `admin` network.  Often times, these single-network-interfaces default to
+an iPXE boot if it can network boot at all.
+
+A suggested course of action in these situations is to, if possible, ALWAYS
+boot using iPXE, but chainloading into a Triton Compute Node boot.
+
+As mentioned in the [iPXE installer](../developer-guide/iso-installer.md),
+one will need to setup an iPXE server in these situations.  Unlike the triton
+installer, there is no tar archive readily available.  The directory,
+however, needs only three files.
+
+
+### An initial .ipxe instruction file.
+
+This file should be inserted into `triton-cn.ipxe`:
+
+```
+#!ipxe
+set base-url https://example.com/triton-cn-ipxe
+kernel ${base-url}/ipxe.lkrn
+module ${base-url}/default-no0.ipxe
+boot
+```
+
+And the URL for the BMaaS iPXE should be, in this example,
+`https://example.com/triton-cn-ipxe/triton-cn.ipxe`.  There are two other
+files mentioned in this, and those are what we need as well.
+
+### An iPXE binary
+
+In a head node, the file `/opt/smartdc/share/usbkey/contents/boot/ipxe.lkrn`
+exists, and is the Triton-special iPXE binary that a Triton Compute Node
```

Comment:
Either 'exists and is the...' or 'exists, and it is the....'

Reply:
The latter of those two, sure.